src/main/java/bhs/devilbotz/commands/arm/ArmSafety.java

```diff
@@ -82,6 +84,9 @@ public final void initialize() {
 
   @Override
   public final void execute() {
+    if (isArmStuck()) {
+      currentCommand = ArmCommand.EMERGENCY_STOP;
+    }
```

Comment:
This section needs to be moved to below the `executeWithSafety()` call.  Otherwise, the child class may override the currentCommand by calling `up()` or `down`.   So, we need to implement all safety checks AFTER the child has been called.